net.go

```diff
@@ -80,6 +81,18 @@ func Dial(network, addr string, ctx *Ctx, flags DialFlags) (*Conn, error) {
 	return DialSession(network, addr, ctx, flags, nil)
 }
 
+// DialTimeout acts like Dial but takes a timeout for network dial.
+//
+// The timeout includes only network dial. It does not include OpenSSL calls.
+//
+// See func Dial for a description of the network, addr, ctx and flags
+// parameters.
+func DialTimeout(network, addr string, timeout time.Duration, ctx *Ctx,
```

Comment:
Why omit the session param?

Reply:
A reason is same as for regular `Dial`:
https://github.com/libp2p/go-openssl/blob/6a218fb8ebfb2586d4653e61fe58565fd372fddf/net.go#L80
There is no need for this.

Reply:
Ah I see. Thanks